README.rst

```diff
@@ -1,52 +1,119 @@
 |Travis Build Status| |Codacy| |Coverage Status| |PyPI|
 
 Adobe Font Development Kit for OpenType (AFDKO)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+===============================================
 
 The AFDKO is a set of tools for building OpenType font files from PostScript and TrueType font data.
 
-This directory tree contains the data files, Python scripts, Perl scripts, and
-sources for the command line programs that comprise the AFDKO. The project uses the `Apache 2.0 OpenSource license <https://rawgit.com/adobe-type-tools/afdko/master/LICENSE.md>`__.
+This repository contains the data files, Python scripts, and sources for the command line programs that comprise the AFDKO. The project uses the `Apache 2.0 OpenSource license`_.
+
+Please refer to the file `AFDKO-Overview.html`_ for a more detailed description of what is included in the package.
+
+.. _Apache 2.0 OpenSource license: https://rawgit.com/adobe-type-tools/afdko/master/LICENSE.md
+.. _AFDKO-Overview.html: https://rawgit.com/adobe-type-tools/afdko/master/afdko/AFDKO-Overview.html
 
-Please refer to the file `AFDKO-Overview.html <https://rawgit.com/adobe-type-tools/afdko/master/afdko/AFDKO-Overview.html>`__ for a more detailed description of what is in the AFDKO.
 
 Major Changes
-~~~~~~~~~~~~~
+-------------
+
+The AFDKO has been restructured so that it can be installed as a Python package. It now depends on the user's Python interpreter, and no longer contains its own Python interpreter.
 
-The afdko has been restructured so that it can be installed as a Python package. It now depends on the user's Python interpreter, and no longer contains its own Python interpreter.
+In order to do this, two Adobe-owned, non-open source programs were dropped: **IS** and **checkOutlines**. If these turn out to be sorely missed, an installer for them will be added to the old Adobe AFDKO website.  The current intent is to migrate the many tests in checkOutlines to the newer **checkOutlinesUFO** (which does work with OpenType and Type 1 fonts, but currently does only overlap detection and removal, and a few basic path checks).
```

Comment:
In order to do … what?
This text probably comes from when the FDK was first open-sourced, the paragraph feels a bit out of context.

Reply:
It's in reference to the previous paragraph. I'll remove the paragraph break.

Reply:
Never mind. I assumed the _In order to do this …_ paragraph was about the original decisions made when the FDK was first open-sourced.